src/app/page/UserProfile/UserOverview.jsx

```diff
@@ -48,18 +50,21 @@ const UserStatus = ({ profile, setProfile, view }) => {
       type: ["(cities)"],
     };
     const addrElem = document.getElementById("address");
+    const googleRef = !!window && !!window.google ? window.google : undefined;
```

Comment:
What's going on here?

Reply:
These should get auto treated as booleans since they're being used to determine the ternary. no need for the double negation.

Also since this is happening after the component has mounted it's safe to assume the window object is already available. Could just say 
```js
window.google ? window.google : undefined;
//or really just
ref = window.google;
// Given that you're only checking for undefined it'll already be undefined if not present
```

Reply:
Addressed in 1ea575f